src/main/java/com/example/bookstoreapp/models/Order.java

```diff
@@ -0,0 +1,40 @@
+package com.example.bookstoreapp.models;
+
+import com.example.bookstoreapp.entities.OrderEntity;
+import lombok.Data;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+@Data
+public class Order {
+
+    private Long orderId;
+    private Long userId;
+    private Date orderdate;
+    private Integer addressId;
+    private BigDecimal totalAmount;
+    private Date deliveryDate;
+    public OrderEntity toEntity() {
```

Comment:
One line break required just to show that it is a method. It will increase readability